domains/enderpoint.json

```diff
@@ -8,6 +8,7 @@
         "instagram": "enderpoint07"
     },
     "record": {
-        "CNAME": "enderpoint07.github.io"
+        "CNAME": "enderpoint07.github.io",
+        "TXT": "zoho-verification=zb07670443.zmverify.zoho.in"
```

Comment:
We don't support TXT verification or allow other records with CNAME records.

Reply:
oh